build.yaml

```diff
@@ -2172,8 +2194,14 @@ steps:
       export HAIL_TEST_STORAGE_URI={{ global.test_storage_uri }}/{{ token }}
       export HAIL_TEST_RESOURCES_DIR="{{ global.test_storage_uri }}/{{ upload_test_resources_to_blob_storage.token }}/test/resources/"
       export HAIL_DOCTEST_DATA_DIR="{{ global.test_storage_uri }}/{{ upload_test_resources_to_blob_storage.token }}/doctest/data/"
+      export HAIL_GENETICS_QOB_VEP_GRCH37_IMAGE={{ qob_vep_image_grch37.image }}
+      export HAIL_GENETICS_QOB_VEP_GRCH38_IMAGE={{ qob_vep_image_grch38.image }}
       export GOOGLE_APPLICATION_CREDENTIALS=/test-gsa-key/key.json
 
+      if [[ "$HAIL_CLOUD" = "gcp" ]] then
```

Comment:
@danking  Can I get some help here for why this code keeps failing?

https://ci.hail.is/batches/6813943/jobs/107

Reply:
Nevermind. Figured it out.